src/main/java/com/fxgraph/edges/CorneredEdge.java

```diff
@@ -93,7 +145,8 @@ public EdgeGraphic(Graph graph, CorneredEdge edge, Orientation orientation, Stri
 					Region target = graph.getGraphic(edge.getTarget());
 					setupArrowIntersect(target, sourceX, targetY, targetX, targetY);
 					group.getChildren().add(arrow);
-				} else {
+				} 
+				else {
```

Comment:
Is this part of your preferred code style or one of the Eclipse defaults?

Reply:
@Col-E It’s just my preference/habit. I can be more conscious to avoid restyling like this in the future, though it seems silly to make another commit to undo this one, especially since there may be more elsewhere.

Reply:
Since these newlines were in the same file as the redundant `CorneredEdge` methods, I just undid them in the same commit https://github.com/Col-E/fxgraph/pull/3/commits/eb76259d75dcf51a1f946cb3758ce1c81c1cc491.